### ISSUE TITLE
how to get correct image sequence for demo?

### ISSUE THREAD

**Issue by akashAD98** [beltoforion/ai_ever_zoom#1]
Error: Image misalignment found! The images may not be in order, try using the "-as" option!

even i tried to use -as argument still giving not matching images warning, can you please add any guidance for this?

```


AI Outpainting Zoom Video Generator
-----------------------------------
 - input folder: "/content/inp"
 - output: "/content/op/vidoinfi.mp4"
 - fps: 60.0
 - zoom factor: 2.0
 - zoom steps: 100
 - zoom crop: 0.8
 - auto sort: True
 - debug aides: False
 - delay: 0.0
 - reverse: False

Reading images from "/content/inp"
 - 14 images read

Determining image order


Image relation matrix:
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --  0.73  --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --  0.73 
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 --   --   --   --   --   --   --   --   --   --   --   --   --   --  
 - found 12 unlinked images.
 - Warning: Your series contains 11 images that cannot be matched!

Finding first image:
 - Discarding image 0 because it is unconnected to other images!
 - Discarding image 3 because it is unconnected to other images!
 - Discarding image 4 because it is unconnected to other images!
 - Discarding image 5 because it is unconnected to other images!
 - Discarding image 6 because it is unconnected to other images!
 - Discarding image 7 because it is unconnected to other images!
 - Discarding image 8 because it is unconnected to other images!
 - Image 9 is the start of a zoom series
 - Discarding image 10 because it is unconnected to other images!
 - Discarding image 11 because it is unconnected to other images!
 - Discarding image 12 because it is unconnected to other images!
 - Image 13 is the start of a zoom series

```

**Comment by beltoforion** [comment]
From the output it appears that either the image sequence is not a proper zoom sequence or the zoom factor was set incorrectly. What images did you use? Without seeing them i cannot say much more.

Midjourney creates sequences with either 2.0 or 1.333 zoom factor. In their UI they say 1.5 instead of 1.33. This is wrong! Use 1.333 if you created the sequence using midjourneys 1.5 button.

**Comment by akashAD98** [comment]
@beltoforion should i need to only use mid-journey images? can you give some reference links to it?

i was directly taking images from lexica, even i tried your video & extracted into frames & passed it but nothing works


where is the option in midjourney to get sequence images? can we get this on automatic1111 or compfy ui?

**Comment by beltoforion** [comment]
I just checked this with sample images from lexica. Lexica is using an outpainting zoom factor of 1.5. You must submit this value to the zf option. It also appears that images created by lexica could not be autosorted reliably.  I have updated the auto sort function and it may work better for lexica images. So if you update your repo it may even work with auto sort.

So i would advise to update the repo and re-run with "-zf 1.5". If auto sort still fails sort the images manually starting with the outermost.
